test/e2e/azure_lb.go

```diff
@@ -120,7 +120,7 @@ func AzureLBSpec(ctx context.Context, inputGetter func() AzureLBSpecInput) {
 		By("creating an internal Load Balancer service")
 
 		ilbService := webDeployment.GetService(ports, deploymentBuilder.InternalLoadbalancer)
-		_, err = servicesClient.Create(ilbService)
+		_, err = servicesClient.Create(context.Background(), ilbService, metav1.CreateOptions{})
```

Comment:
why not pass in the ctx param here?

Reply:
same question for the other test files in e2e

Reply:
yes, it should use ctx